RUFAS/biophysical/animal/animal_grouping_scenarios.py

```
@@ -35,25 +29,24 @@ class AnimalGroupingScenario(Enum):
         AnimalCombination.LAC_COW: [AnimalType.LAC_COW],
     }
 
-    def __init__(self, value: Dict[AnimalCombination, List[str]]):
+    def __init__(self, value: dict[AnimalCombination, list[str]]):
```

Ruff (0.15.15)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)